tests/test_win_writer.py

```diff
@@ -1,18 +1,14 @@
 # -*- coding: utf-8 -*-
-
 from __future__ import absolute_import
 
-import sys
-
 from gaas_sample import create_gaas_win_params
 
 
-def test_create_win_string(configure, create_gaas_win_params, compare_equal):
+def test_create_win_string(configure, create_gaas_win_params, file_regression):
     from aiida_wannier90.io._write_win import _create_win_string
 
-    # The 'tag' is an ugly workaround because win_writer changes
-    # behavior between Python versions 2 and 3.
```

Comment:
I think we still need to figure out some way to work around the differences between Py2 and Py3 here - maybe change the extension to `.py2.win` or `.py3.win` depending on the version?

Reply:
Ah sorry, I didn't see your comment about this. For some reason though, this test fails on the Python 2 travis build.

Reply:
Aha, I think in Python 3 something is automatically converted to a float where it should be an int. See the difference between the deleted and current test files in the `l=...` and `mr=...` part of the projections.

Reply:
Indeed, we are now having a custom conversion for floats where needed, using standard formatting for the rest (integers), I'm going to push changes soon